documentation/rom_simulation/0d-solver/user-guide-run/readme.md

```diff
@@ -1,13 +1,13 @@
 ### Run svZeroDSolver from the command line
 
 svZeroDSolver can be executed from the command line using a JSON configuration
-file.
+file. In this case we will use the test case ```steadyFlow_RLC_R.json```. Once you have cloned the svZeroDSolver Github repository and built svZeroDSolver, navigate to the top level of the svZeroDSolver repository and run the following command.
 
```

Comment:
You should not have to give instructions about building the 0D solver here, assume it has been installed.